src/shared/route-urls.ts

```diff
@@ -36,11 +38,11 @@ export enum RouteUrls {
   IncreaseStxFee = '/increase-fee/stx',
   IncreaseBtcFee = '/increase-fee/btc',
   IncreaseFeeSent = '/increase-fee/sent',
```

Comment:
nested routes must have relative paths so I had to change these. 

I can investigate un-nesting `stx` and `btc` as it may make for cleaner re-directs elsewhere